ui/helpers/utils/util.test.js

```diff
@@ -194,6 +195,37 @@ describe('util', () => {
     });
   });
 
+  describe('#getIsBrowserDeprecated', () => {
```

Comment:
Could you add test cases for Edge and Opera as well?

I'm also curious how browser forks are treated here. Could you add a test for an unrecognized browser?

Reply:
Right now it looks like Opera is unhandled because bowser handles by name, but I've manually tested version 58 which was released Jan/23/2019 (around the same time as Chromium v78 and it seemed to work fine. I can add it to the list of outdated browser ranges as '<58'

Reply:
Could you clarify what you mean by "unhandled"? Does that mean it will _always_ say that the browser is outdated, or `_never_`? Not sure what the default case is.

Reply:
The default case is the current functionality, where the user would not see anything different. So it will never say the browser is outdated if it's not in [this constant](https://github.com/MetaMask/metamask-extension/blob/938fd4ebaca5310c8a1b7b94214d11e383edc7c1/ui/helpers/constants/common.js#L27-L30)

Reply:
The Wikipedia page claims that Opera 65 was based on Chromium v78, so maybe we can set that as the minimum for Opera: https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser

Reply:
Oh sorry, it's Chrome 80 we're targeting, which'd be Opera 67

>Opera 67 was released on February 25, 2020, based on Chromium 80.